### ISSUE TITLE
Build fails on Linux

### ISSUE THREAD

**Issue by callmeStriking** [taylor001/crown#51]
Arch Linux 4.3.3, GCC 5.3.0

Log: 

```
[rob@Archie][~/dev/crown]% GENIE=`which genie` make linux-release64 -j128
/usr/bin/genie --file=genie/genie.lua --with-luajit --with-openal --with-bullet --with-tools --compiler=linux-gcc gmake
Building configurations...
Running action 'gmake'...
Generating build/projects/linux/Makefile...
Generating build/projects/linux/bgfx.make...
Generating build/projects/linux/shaderc.make...
Generating build/projects/linux/openal.make...
Generating build/projects/linux/bullet.make...
Generating build/projects/linux/crown.make...
Done.
make -R -C build/projects/linux config=release64
make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
make[1]: Entering directory '/home/rob/dev/crown/build/projects/linux'
==== Building bgfx (release64) ====
==== Building shaderc (release64) ====
==== Building openal (release64) ====
==== Building bullet (release64) ====
==== Building crown (release64) ====
sound_world_al.cpp
../../../src/audio/sound_world_al.cpp:11:22: fatal error: id_array.h: No such file or directory
compilation terminated.
crown.make:1039: recipe for target '../../linux64/obj/x64/release/crown/src/audio/sound_world_al.o' failed
make[2]: *** [../../linux64/obj/x64/release/crown/src/audio/sound_world_al.o] Error 1
Makefile:32: recipe for target 'crown' failed
make[1]: *** [crown] Error 2
make[1]: Leaving directory '/home/rob/dev/crown/build/projects/linux'
makefile:38: recipe for target 'linux-release64' failed
make: *** [linux-release64] Error 2
```

It looks like a file is entirely missing from the repo.

**Comment by taylor001** [comment]
Yup, I'm rewriting that stuff. This should fix the build: a28185d1ee7d5884a20d382ef2e9572d53052323

**Comment by callmeStriking** [comment]
It works! Although `make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.` is annoying, because I can't compile in parallel without trickery.
